Source/WebCore/platform/graphics/cocoa/FontCacheCoreText.cpp

```diff
@@ -967,21 +967,34 @@ void FontCache::prewarmGlobally()
     if (MemoryPressureHandler::singleton().isUnderMemoryPressure())
         return;
 
-    Vector<String> families {
-#if PLATFORM(MAC) || PLATFORM(MACCATALYST)
```

Comment:
This change looks fine, but I'm not sure about removing these items. Should we check with Myles?

Reply:
We are not prewarming fonts anymore on macOS, due to the way we block the font daemon, so I believe this change should be ok.

Thanks for reviewing!